builder/commands_test.go

```diff
@@ -0,0 +1,27 @@
+package builder
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Masterminds/cookoo"
+)
+
+func TestSleep(t *testing.T) {
+	reg, router, cxt := cookoo.Cookoo()
+
+	reg.Route("test", "Test route").
+		Does(Sleep, "res").Using("duration").WithDefault(3 * time.Second)
+
+	start := time.Now()
+	err := router.HandleRequest("test", cxt, true)
+	if err != nil {
```

Comment:
Done. I'm not a huge stickler for it, but I think there is good reason to prefer scoping vars to a smaller block. So, I think it makes (performance) sense to do this where possible. It's not just stylistic.